src/components/EntityInfo.vue

```diff
@@ -266,9 +271,15 @@ export default class EntityInfo extends Vue {
     return sessionStorage.getItem('TEMP_REG_NUMBER')
   }
 
+  /** The limited restoration active-until date, if it exists, otherwise null. */
+  get activeUntil (): string {
+    const date = this.yyyyMmDdToDate(this.getStateFiling?.restoration?.expiry)
+    return this.dateToPacificDate(date, true)
+  }
+
```

Comment:
What do you think about moving this method to a store getter?  At the very least, I think it would be wise to have a getter for the expiry date -- `this.getStateFiling?.restoration?.expiry`.   If we change the getStateFiling API response in the future, having this method as a getter allows us to easily refactor.  While this logic exists in a component, it's more difficult to find.

Reply:
Also, what do you think about bringing the `|| 'Unknown` that exists in the template back into `activeUntil()`?

Reply:
> What do you think about moving this method to a store getter?

Done.